web-platform-core/Gladstone/Gladstone.js

```diff
@@ -0,0 +1,81 @@
+const gladstone = (database = 'Gladstone') => {
```

Comment:
```suggestion
// SPDX-License-Identifier: Apache-2.0

const gladstone = (database = 'Gladstone') => {
```

Reply:
The above should be good enough. I see that we have a LICENSE file at the root of this repo, which should cover this folder, as well.